Classes/TYPO3/Hooks/BackendAjaxHook.php

```diff
@@ -54,7 +50,17 @@ public function banAll(ServerRequestInterface $request, ResponseInterface $respo
             );
         }
 
-        return json_encode($response);
+        // We need to return a response to satisfy the
+        // TYPO3\CMS\Backend\Http\RouteDispatcher::dispatch()
+        // don't like the solution, but haven't come up with something better yet.
+        return new Response();
+    }
+
+    protected function getVarnish(): Varnish
+    {
+        $objectManager = GeneralUtility::makeInstance(ObjectManager::class);
+        $this->varnish = $objectManager->get(Varnish::class);
```

Comment:
I know that the ObjectManager->get() is deprecated, but haven't found a better way to do this in context of a Backend Hook. The GeneralUtility::makeInstance() isn't working in this context for the Varnish instance.